plugins/rest/Dockerfile

```diff
@@ -1,29 +1,20 @@
-FROM rapid7/insightconnect-python-3-38-slim-plugin:4
-# The three supported python parent images are:
-# - komand/python-2-plugin
-# - komand/python-3-plugin
-# - komand/python-pypy3-plugin
-#
-LABEL organization=komand
+FROM rapid7/insightconnect-python-3-38-plugin:4
 
-# Add any custom package dependencies here
-# NOTE: Add pip packages to requirements.txt
+LABEL organization=rapid7
+LABEL sdk=python
 
-# End package dependencies
-
-# Add source code
 WORKDIR /python/src
+
 ADD ./plugin.spec.yaml /plugin.spec.yaml
-ADD . /python/src
+ADD ./requirements.txt /python/src/requirements.txt
 
-# Install pip dependencies
 RUN if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
 
-# Install plugin
-RUN python setup.py build && python setup.py install
+ADD . /python/src
+
+RUN	python setup.py build && python setup.py install
 
-# needed for SSL Verify
-USER root
+# User to run plugin code. The two supported users are: root, nobody
+USER nobody
 
-# Entrypoint
-ENTRYPOINT ["/usr/local/bin/komand_rest"]
+ENTRYPOINT ["/usr/local/bin/icon_rest"]
```

Comment:
Trailing New Line
```suggestion
ENTRYPOINT ["/usr/local/bin/icon_rest"]

```

Reply:
I'm not sure if the trailing new line actually affects much, but we tend to leave them in where we can, could you add them back where the tooling removed it?

Reply:
It shouldn't be updated. The correct `ENTRYPOINT` here is `komand_rest` 😄